v1.18/k3s/README.md

```diff
@@ -0,0 +1,14 @@
+# k3s
+Lightweight Kubernetes
+## To Reproduce
+
+1. Create two machines, any modern Linux will work but this test was done with Ubuntu 20.04
+2. On machine one download k3s v1.18.4+k3s1 and start the server
```

Comment:
@ShylajaDevadiga if there is a single install command via curling the install script, perhaps you could include that here

Reply:
Updated README to include curl script.